main_test.go

```diff
@@ -80,12 +84,20 @@ func TestAcceptance(t *testing.T) {
 		`memcached_up 1`,
 		`memcached_commands_total{command="get",status="hit"} 2`,
 		`memcached_commands_total{command="get",status="miss"} 1`,
-		`memcached_commands_total{command="set",status="hit"} 2`,
+		`memcached_commands_total{command="set",status="hit"} 3`,
 		`memcached_commands_total{command="cas",status="hit"} 1`,
-		`memcached_current_bytes 74`,
+		`memcached_current_bytes 274`,
 		`memcached_current_connections 11`,
-		`memcached_current_items 1`,
-		`memcached_items_total 3`,
+		`memcached_current_items 2`,
+		`memcached_items_total 4`,
+		`memcached_slab_current_items{slab="1"} 1`,
+		`memcached_slab_current_items{slab="5"} 1`,
+		`memcached_slab_commands_total{command="set",slab="1",status="hit"} 2`,
+		`memcached_slab_commands_total{command="cas",slab="1",status="hit"} 1`,
+		`memcached_slab_commands_total{command="set",slab="5",status="hit"} 0`,
```

Comment:
I would think so too but when running the test suite it passes. I haven't really dug into why there isn't a hit. Maybe because the test doesn't try to retrieve the key?